modules/drivers/mongo/test/metabase/driver/mongo/execute_test.clj

```diff
@@ -53,3 +67,69 @@
                            [1 "Rasta"     now nil]]
                     :columns ["_id" "name" "last_login" "alias"]}
                    (mt/rows+column-names (qp/process-query query))))))))))
+
+(defn- interrupt-ex? [^MongoCommandException ex]
+  (and (= 11601 (.getErrorCode ex))
+       (= "Interrupted" (.getErrorCodeName ex))))
+
+(deftest kill-an-in-flight-query-method-validation-test
+  (testing "Confirm that method used to kill in-flight queries on mongo works"
```

Comment:
I wonder if it's worth having this test. If it fails, the next test should fail too. I can help debugging maybe, but I'm not sure that's worth the maintenance effort.

Reply:
I'm removing it. I've used it during the development, while trying to wire up sessions with our then current mongo code. Now it's obsolete.